### PR TITLE
fix: remove grid and reset styles from config defaults

### LINKED ISSUE #2762: Using vuestic-ui in new nuxt project breaks layout
**Vuestic-ui version:** 1.5.2

imageimage

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Close #2762

## Description
- Removed grid and reset stylesheets from pre-defined configuration.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by asvae** [comment]
Please also be careful and be sure this does not introduce breaking change.

File: packages/nuxt/src/module.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default defineNuxtModule<VuesticOptions>({
 
   defaults: {
     config: {},
-    css: true,
+    css: ['smart-helpers', 'typography'],
```
Comment (m0ksem) [inline packages/nuxt/src/module.ts:23]: Also, update `@default true` to `@default ['smart-helpers', 'typography']` in types.

**Review by m0ksem — APPROVED** [review]
I fixed nuxt build, so you can test it.

File: packages/docs/modules/vuestic.ts [hunk 2]
```diff
@@ -25,8 +25,10 @@ export default defineNuxtModule<VuesticOptions>({
     nuxt.options.alias['@vuestic/ag-grid-theme'] = resolve(__dirname, '../../ag-grid-theme/src/styles/index.scss');
     nuxt.options.alias['vuestic-ui/styles/typography.css'] = resolve(__dirname, '../../ui/src/styles/typography/typography.scss');
     nuxt.options.alias['vuestic-ui/styles/grid'] = resolve(__dirname, '../../ui/src/styles/grid/grid.scss');
+    nuxt.options.alias['vuestic-ui/styles/grid.css'] = resolve(__dirname, '../../ui/src/styles/grid/grid.scss');
```
Comment (aluarius) [inline packages/docs/modules/vuestic.ts:28]: Can't understand, why we need these two aliases.

**Review by aluarius — APPROVED** [review]

File: packages/docs/modules/vuestic.ts (hunk 2 above)
Comment (m0ksem) [reply packages/docs/modules/vuestic.ts:28]: Same. But it doesn't look harmful.

Comment (rustem-nasyrov) [reply packages/docs/modules/vuestic.ts:28]: Otherwise docs build fails.

**Comment by m0ksem** [comment]
I'm gonna release 1.1.0 after this PR.

**Pull request merged**